packages/cardano-services/src/Program/services/postgres.ts

```diff
@@ -17,7 +17,15 @@ import fs from 'fs';
 const timedQuery = (pool: Pool, logger: Logger) => async (args: string | QueryConfig, values?: any) => {
   const startTime = Date.now();
   const result = await pool.query(args, values);
-  logger.debug(`Query\n${args}\ntook ${Date.now() - startTime} milliseconds`);
+  const query =
+    typeof args === 'string'
+      ? `${args} ${JSON.stringify(values)}`
+      : 'text' in args && typeof args.text === 'string'
+      ? `${args.text} ${JSON.stringify(args.values)}`
+      : JSON.stringify({ args, values });
```

Comment:
🎨 next time please use `if` `else`

Reply:
I personally find this perfectly readable :sweat_smile:

Reply:
I agree, this is cognitive overhead. Our lint rules already prevent it, but it's been disabled in the file 💀 

https://github.com/input-output-hk/cardano-js-sdk/pull/762/files/03969eb59f02166983db51e734ecbf677393c0b2#diff-3df69ebf26a058dd3fccae9c81477a3a06fdab0ed5cfe837f8c26ddde64f07f2R3